app/controllers/concerns/with_user_params.rb

```diff
@@ -6,6 +6,6 @@ def user_params
   end
 
   def permissible_params
-    User.profile_fields
+    User.profile_fields + [:avatar_id, :image_url]
```

Comment:
Want to hear your thoughts on this @luchotc. I'd rather not add these here, but the form ignores those fields unless we tell them they're permissible.

Reply:
We should add them only in the appropriate controller, as this one is used in several.
I think in users_controller.rb we should do something like:

```
def permissible_params
    super + [:avatar_id, :image_url]
end
```